<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-CBTS99R" failures="2" tests="16" name="testRunner.runners" time="214.633" errors="0" timestamp="2023-02-03T22:30:46 EET" skipped="0">
  <testcase classname="testRunner.runners" name="runScenario" time="8.241"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="26.668"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="11.659"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="18.315"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="19.582"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="9.714"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="9.615"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="11.158">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefs.D06_externalLinks.link_opens_in_new_tab(D06_externalLinks.java:23)
at ✽.link opens in a new tab(file:///F:/TestingPractice/SeleniumPrac/src/main/resources/features/F06_externalLinks.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="10.859"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="9.514"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="12.602"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="8.489">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefs.D04_search.should_appear_in_the_search_results(D04_search.java:24)
at ✽."book" should appear in the search results(file:///F:/TestingPractice/SeleniumPrac/src/main/resources/features/F04_search.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="20.685"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="8.757"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="10.461"/>
  <system-out/>
  <testcase classname="testRunner.runners" name="runScenario" time="18.314"/>
  <system-out/>
</testsuite> <!-- testRunner.runners -->
